src/App.vue

```diff
@@ -63,8 +63,35 @@ export default {
                 },
             ],
 
+            searchTerm: "",
+            selectedHashtag: null,
+
         };
     },
+    computed: {
+        filteredBlogDatas() {
+            if (this.selectedHashtag) {
+                return this.blogDatas.filter((blogData) =>
+                    blogData.hashtags.includes(this.selectedHashtag)
+                );
+            } else {
+                return this.blogDatas.filter((blogData) =>
+                    blogData.hashtags.some((hashtag) =>
+                        hashtag.includes(this.searchTerm)
+                    )
+                );
+            }
+        },
+    },
+    methods: {
+        increaseLike(id) {
+            const blogData = this.blogDatas.find((data) => data.id === id);
+            blogData.like++;
+        },
+        searchHash() {
```

Comment:
is this function need `searchHash `? because this is only assigning a value from 1 variable to another.